src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import exceptions.DukeException;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] taskList = new Task [100];
+    static int taskIndex=0;
```

Comment:
According to the Java coding standard, there should be whitespaces before and after the equal sign.